pkg/i18n/i18n.go

```diff
@@ -61,18 +63,10 @@ func (l *Localizer) GetLanguage() string {
 
 // add translation file(s)
 func addBundles(log *logrus.Logger, i18nBundle *i18n.Bundle) {
-	fs := []func(*i18n.Bundle) error{
-		addPolish,
-		addDutch,
-		addEnglish,
-	}
-
-	for _, f := range fs {
-		if err := f(i18nBundle); err != nil {
-			log.Fatal(err)
-
-		}
-	}
+	i18nBundle.RegisterUnmarshalFunc("toml", toml.Unmarshal)
```

Comment:
because this code runs at runtime rather than compile time, won't this only work if the use has the project checked out rather than using a binary?

Reply:
The files will need to be shipped with the binary just like regular langpacks. No idea what the convention for Go is.